bedrock/firefox/templates/firefox/new.html

```diff
@@ -33,7 +33,6 @@
   <h2>{{ high_res_img('img/firefox/new/header-firefox.png', {'alt': _('Firefox for desktop'), 'width': '223', 'height': '84'}) }}</h2>
 {% endblock %}
 
-{% block site_css %}{% endblock %}
```

Comment:
This emptying of the `site_css` block needs to remain, otherwise modern browsers get served duplicate CSS.

This is the last thing :exclamation: